package.json

```diff
@@ -45,25 +45,35 @@
     "webpack-merge": "5.10.0"
   },
   "dependencies": {
+    "bootstrap-icons": "^1.11.3",
+    "buffer": "^6.0.3",
     "date-fns": "^2.29",
+    "dotenv": "^16.4.5",
+    "dotenv-webpack": "^8.1.0",
+    "highlight.js": "^11.10.0",
     "isomorphic-fetch": "^3.0.0",
+    "JSONStream": "^1.3.5",
     "linkify-react": "^3.0.4",
     "lodash": "^4.17.21",
     "prop-types": "^15.7.2",
     "query-string": "^7.1.3",
     "react": "^16.14.0",
     "react-bootstrap": "^2.7.2",
     "react-dom": "^16.14.0",
+    "react-highlight": "^0.15.0",
     "react-html-parser": "^2.0.2",
     "react-icons": "^4.3.1",
+    "react-markdown": "^8.0.7",
     "react-nl2br": "^1.0.2",
     "react-redux": "^7.2.6",
     "react-router": "^3.2.6",
+    "react-select": "5.7.3",
     "react-table": "^7.7.0",
     "react-window": "^1.8.8",
     "redux": "^4.2.1",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.4.1",
-    "react-select": "5.7.3"
+    "remark-gfm": "^3.0.1",
+    "stream-browserify": "^3.0.0"
```

Comment:
`stream-browserify`, `JSONStream` and `buffer` are no longer necessary.

I would also propose to remove `bootstrap-icons`. For this purpose we already have `react-icons` (https://react-icons.github.io/react-icons/) which you should use instead. Consider to use the FontAwesome Set there, as this is what I was using for everything else (with a few minor exceptions) to keep things consistent.